scripts/autorelease.sh

```diff
@@ -43,4 +41,4 @@ for file in $assets_dir/*; do
 done
 
 # Create release
-hub release create "${assets[@]}" -F "$TEMPDIR"/"$project"/*-release-notes.md --prerelease auto-release
+hub release create "${assets[@]}" -F "$TEMPDIR"/"$project"/release-notes.md --prerelease auto-release
```

Comment:
Have you tested releasing, btw?

Reply:
No.

Reply:
Tested a release, see here: https://github.com/serokell/tezos-packaging/releases/tag/auto-release

Reply:
![image](https://user-images.githubusercontent.com/14182218/78813012-0eb59e80-79d5-11ea-8d23-969d17a7c7af.png)
looks weird, I won't say that `006-PsCARTHA` binaries target the 005 protocol

Reply:
Well, this is just wording. Binaries that don't have a suffix do indeed target 005 and 006. This is expressed in the updated README.

Reply:
Updated the changelog.